gitlab/gitlabclient.go

```diff
@@ -409,6 +409,7 @@ func (c *gitlabClientImpl) ListCommitsPage(ctx context.Context, projectName stri
 			AuthorName: c.AuthorName,
 			Message:    c.Message,
 			CreatedAt:  c.CreatedAt,
+			URL:        c.WebURL,
```

Comment:
It seems that gitlab.Commit does not have and URL field but WebURL... So I guess you can revert this to your initial implem. Sorry for the trouble.